Upstream-Status: Backport

From b7578c6b7bd966b63ab1b2682fd90ed4f3a92e71 Mon Sep 17 00:00:00 2001
From: Ian Lance Taylor <ian@airs.com>
Date: Fri, 6 Apr 2012 17:23:58 +0000
Subject: [PATCH 144/262] 	* timer.cc: #include <unistd.h>.

---
 gold/ChangeLog |    4 ++++
 gold/timer.cc  |    2 ++
 2 files changed, 6 insertions(+)

2012-04-06  Ian Lance Taylor  <iant@google.com>

	* timer.cc: #include <unistd.h>.

diff --git a/gold/timer.cc b/gold/timer.cc
index d9b8874..44e19f5 100644
--- a/gold/timer.cc
+++ b/gold/timer.cc
@@ -22,6 +22,8 @@
 
 #include "gold.h"
 
+#include <unistd.h>
+
 #ifdef HAVE_TIMES
 #include <sys/times.h>
 #endif
-- 
1.7.9.5

